scripts/EN/strtab_script

```diff
@@ -1,849 +1,849 @@
+618: ¥"The unfinished diary¥"
+619: ¥"Thinking of Alessa¥"
+620: ¥"The woman in a nurse uniform¥"
+621: ¥"Lisa throws her arms around Harry¥"
+622: ¥"A real or a false image?¥"
+623: ¥"Let me ask you…¥"
+624: ¥"Lisa has vanished¥"
+625: ¥"The other church¥"
+626: ¥"Harry runs away¥"
+627: ¥"Readying the gun to the rear¥"
+628: ¥"Looking inside the cabinet¥"
+629: ¥"Looking on top of the cabinet¥"
+630: ¥"Picking up the picture¥"
+631: ¥"Harry points the gun at Lisa¥"
+632: ¥"A reunion with Cybil¥"
+633: ¥"Relief¥"
+634: ¥"About Dahlia¥"
+635: ¥"Who should go first?¥"
+636: ¥"Harry says ¥"I'm going¥"¥"
+637: ¥"The other church¥"
+638: ¥"The voice calling from the darkness¥"
+639: ¥"An important piece of information¥"
+640: ¥"Lisa is…¥"
+641: ¥"Cybil says ¥"I'm going¥"¥"
+642: ¥"Cheryl appears on the monitors¥"
+643: ¥"The building appears to be in ruins¥"
+644: ¥"The monitors in the hall¥"
+645: ¥"From where…¥"
+646: ¥"Searching the second floor¥"
+647: ¥"A caterpillar with fierce tusks¥"
+648: ¥"Acquiring the rifle¥"
+649: ¥"My best option is?¥"
+650: ¥"Harry fires the rifle¥"
+652: ¥"A shaky strategy¥"
+653: ¥"Harry moves onto the steel plate¥"
+654: ¥"A sign of the unseen enemy¥"
+655: ¥"Guessing¥"
+656: ¥"On the steel plate…I sense it from behind¥"
+658: ¥"Searching the first floor¥"
+659: ¥"On the steel plate…I sense it in front of me¥"
+660: ¥"Sandy soil…I sense it behind me¥"
+661: ¥"Sandy soil…I sense it in front of me¥"
+662: ¥"On the steel plate…I sense it at my side¥"
+663: ¥"Harry fights where he was standing¥"
+664: ¥"Guessing¥"
+665: ¥"Harry decides that it is best to run away¥"
+666: ¥"A reunion with Lisa¥"
+667: ¥"Lisa rushes to Harry¥"
+668: ¥"Lisa says ¥"Don't go¥"¥"
+669: ¥"Harry says ¥"Come with me¥"¥"
+670: ¥"The matured monster¥"
+671: ¥"I must kill it here…¥"
+672: ¥"Firing the rifle while avoiding it¥"
+674: ¥"The unseen weak point¥"
+675: ¥"The target is its abdomen¥"
+677: ¥"Harry says ¥"I can't¥"¥"
+678: ¥"The target is its wings¥"
+679: ¥"The target is its head¥"
+680: ¥"Hiding behind the water tower¥"
+681: ¥"Sounds of a struggle¥"
+682: ¥"Kaufmann is attacked¥"
+683: ¥"After the battle¥"
+684: ¥"The question that had been on my mind¥"
+685: ¥"About Alessa¥"
+686: ¥"Kaufmann's secret¥"
+687: ¥"Garage¥"
+688: ¥"The man who's face turned white with anger¥"
+689: ¥"Showing Kaufmann the picture of Alessa¥"
+690: ¥"On the small boat¥"
+691: ¥"Despair¥"
+692: ¥"A familiar smile¥"
+693: ¥"Dahlia's warning¥"
+694: ¥"Cybil and Harry split up¥"
+695: ¥"Harry heads to the lighthouse¥"
+696: ¥"Harry heads to the amusement park¥"
+697: ¥"Facing Alessa¥"
+698: ¥"The crest on the floor¥"
+699: ¥"The girl disappeared¥"
+700: ¥"Giving up and returning to the boat¥"
+702: ¥"Just where did that girl go?¥"
+703: ¥"The illusion reflected in the spatial distortion¥"
+704: ¥"Jumping into the distortion¥"
+705: ¥"Harry senses danger and turns back¥"
+707: ¥"Rescue of transformed Cybil¥"
+708: ¥"Cybil, from behind¥"
+709: ¥"Plastic Bottle¥"
```

Comment:
Changed pet to plastic to match the PS1 version. (PET refers to the type of plastic and is how they refer to these bottles in Japan.)